spec/controllers/graphql_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+RSpec.describe GraphqlController do
```

Comment:
This is a controller spec. In general request specs are now preferred over controller specs, because they're more real. Controller specs don't actually run HTTP requests, so there's some stuff that might work in a controller test but wouldn't in the real world.

@remcohh what's your opinion on this?

Reply:
I talked about this with @remcohh and he agreed with you that request specs are preferred.